DESCRIPTION

```diff
@@ -80,6 +78,7 @@ Imports:
     mime (>= 0.3),
     jsonlite (>= 0.9.16),
     xtable,
+    fontawesome,
```

Comment:
It would be good to use the upcoming CRAN version as a requirement, in case users have the old version installed.

Reply:
Minimum version requirement has been added here. Thanks!